include/powerloader/mirror.hpp

```diff
@@ -62,11 +62,29 @@ namespace powerloader
         }
     };
 
+    inline std::string strip_trailing_slash(const std::string& s)
```

Comment:
This is implementation detail and should not be in the header. If ued by other units, the declaratoin should be moved in `utils.hpp` and the implementation in `utils.cpp`